services/core-api/app/api/verifiable_credentials/manager.py

```diff
@@ -81,8 +83,9 @@ class W3CCred(BaseModel):
     credentialSchema: List[dict]
 
 
-def convert_date_to_iso_datetime(date: date) -> str:
-    return datetime(date.year, date.month, date.day, 0, 0, 0, tzinfo=ZoneInfo("UTC")).isoformat()
+def convert_date_to_iso_datetime(datetime: datetime) -> str:
+    return datetime(
+        datetime.year, datetime.month, datetime.day, 0, 0, 0, tzinfo=ZoneInfo("UTC")).isoformat()
```

Comment:
Have to check, just because we have a million issues with UTC times- this _converts_ the datetime to UTC, not tells it that the timezone it has is UTC, right?

Reply:
datetime objects are natively timezone-unaware. 

the python language is deprecating some of these methods in favour or producting timezone-aware datetime objects

well formed VCDM credentials required timezone codes, i could just append `T00:00:00` to the string, but this is a more pythonic way to do that and is better understood by future devs

Reply:
What that means is existing datetime objects based on UTC don't know they are based on UTC, they are date+time with no timezone information. it's up to the code to know that it's based on UTC, or maybe postgres is automatically adding UTC information? 

it's probably something that should become it's own task. to update the python to start using the new timezone-aware methods and classes, update the validators to require timezone-aware objects (or add it), then update remaining datetime values to have timezone information.